doc/dev/tests.md

```diff
@@ -15,11 +15,10 @@ In this document we will provide the introduction to the testing framework by:
 
 ## Setup your development environment
 
-The Azure SDK Python team creates libraries that are compatible with Python 2.7 and 3.5 and up. We will set up working Python virtual environments for Python 2.7, 3.5, and 3.9. It is recommended to do your development work in Python3, however it is helpful to have virtual environments for other versions to make debugging PRs easier locally.
+The Azure SDK Python team creates libraries that are compatible with Python 3.6 and up. We will set up working Python virtual environments for Python 3.6, and 3.9. It is recommended to do your development work in Python3, however it is helpful to have virtual environments for other versions to make debugging PRs easier locally.
```

Comment:
Is there a specific reason we don't recommend Python 3.10?

Reply:
Not specifically no.